docs/manual-EN/1.overview/2.quick-start/1.get-started.md

```diff
@@ -1,306 +1,414 @@
+
+```
+
+### Fetching Data
+
+After you insert some data in **Nebula Graph**, you can retrieve any data from your graph database.
+
+To fetch the data of the player whose vertex ID is `100`, enter the following statement:
+
+```ngql
+nebula>FETCH PROP ON player 100;
+```
+
+The following information is displayed:
 
-Insert the vertices and edges based on the graph above.
+```
+============================
+| player.name | player.age |
+============================
+| Tim         | 42         |
+----------------------------
+```
+
+To fetch the data of the _serve_ edge between vertex 100 (100 is the ID of this vertex) and vertex 200 (200 is the ID of this vertex), enter the following statement:
 
 ```ngql
--- Insert vertices
-nebula> INSERT VERTEX student(name, age, gender) VALUES 200:("Monica", 16, "female");
-nebula> INSERT VERTEX student(name, age, gender) VALUES 201:("Mike", 18, "male");
-nebula> INSERT VERTEX student(name, age, gender) VALUES 202:("Jane", 17, "female");
-nebula> INSERT VERTEX course(name, credits),building(name) VALUES 101:("Math", 3, "No5");
-nebula> INSERT VERTEX course(name, credits),building(name) VALUES 102:("English", 6, "No11");
+nebula>FETCH PROP ON serve 100 -> 200;
+```
+
+The following information is displayed:
 
--- Insert edges
-nebula> INSERT EDGE choose(grade) VALUES 200 -> 101:(5);
-nebula> INSERT EDGE choose(grade) VALUES 200 -> 102:(3);
-nebula> INSERT EDGE choose(grade) VALUES 201 -> 102:(3);
-nebula> INSERT EDGE choose(grade) VALUES 202 -> 102:(3);
-nebula> INSERT EDGE follow(likeness) VALUES 200 -> 201:(92.5);
-nebula> INSERT EDGE follow(likeness) VALUES 201 -> 200:(85.6);
-nebula> INSERT EDGE follow(likeness) VALUES 201 -> 202:(93.2);
 ```
+=====================================
+| serve.start_year | serve.end_year |
+=====================================
+| 1997             | 2016           |
+-------------------------------------
+```
+
+### Updating Data
+
+You can update the vertices and edges you just inserted.
+
+#### Updating Vertices
+
+The following example shows you how to change the _age_ value of vertex `100` from `42` to `41`.
 
-### Update Data
+Enter the following statement to update the _age_ value:
 
 ```ngql
--- Update vertex
-nebula> UPDATE VERTEX 101 SET course.credits = $^.course.credits + 1, building.name = "No8" YIELD $^.course.name AS Name, $^.course.credits AS Credits, $^.building.name
--- Update edge
-nebula> UPDATE EDGE 200 -> 101 OF choose SET grade = choose.grade +1
+nebula> UPDATE VERTEX 100 SET player.age = $^.player.age - 1
+```
+
+**Note**: `$^` represents the source vertex.
+
+To check whether the _age_ value is updated, enter the following statement:
+
+```ngql
+nebula> FETCH PROP ON player 100;
+```
+
+The following information is displayed:
+
+```
+============================
+| player.name | player.age |
+============================
+| Time        | 41         |
+----------------------------
 ```
```

Comment:
Besides, you can also 

``ngql
nebula> UPDATE VERTEX 100 SET player.age = $^.player.age - 1
```